pylintrc

```diff
@@ -0,0 +1,29 @@
+[MESSAGES CONTROL]
+disable = 
+	locally-disabled,
+	locally-enabled,
+	no-init,
+	no-self-use,
+	too-few-public-methods,
+	too-many-ancestors,
+	too-many-arguments,
+	too-many-instance-attributes,
+	too-many-public-methods,
```

Comment:
It's silly to just have to end up locally disabling these directly in all of the XBlock `.py` files;
let's just always ignore them outright.